### DISCUSSION TITLE
Allow to turn off features from the main view

### DISCUSSION THREAD

**Discussion by MrChocolatine** [mullvad/mullvadvpn-app#8994] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

I would like to be able to turn off any active features displayed on the main view.

For instance, from this main view:

<img width="432" alt="main view" src="https://github.com/user-attachments/assets/b1c46f80-e409-42b6-b176-b329e738dca7">

It would be nice to be able to click on `DAITA` or `Local network sharing` or (...) to turn these feature off.
Of course, this would only be possible for features that have 2 states: ON and OFF.

To turn these off, maybe we could click their label. Or, during a mouse over, a ❌ could show up on their right.

I think this addition could really improve the user experience, so we don't need to navigate several views to turn off/modify features currently enabled.

### Alternative solutions

To be automatically redirected to the corresponding setting menu when clicking those labels => #7134

### Type of feature

- [ ] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [X] Easier to use
- [ ] Other

### Operating System

- [x] Android
- [x] iOS
- [X] Windows
- [X] macOS
- [X] Linux

**Comment by MarkusPettersson98** [comment]
Hello! :wave: 

Clickable feature indicators (i.e. your alternative solution) will be part of the next beta :tada:

I'll go ahead and close this discussion :blush:

**Reply by MrChocolatine** [reply]
That's great. You rock guys.
